src/test/groovy/org/araneforseti/boundary/fields/EmailFieldTest.groovy

```diff
@@ -53,4 +57,9 @@ class EmailFieldTest {
         scenario_messages_contains(messageName, optionalField)
         scenario_messages_contains(messageName, requiredField)
     }
+
+    @Test
+    void field_uses_default_messageConfiguration() {
```

Comment:
Only the defaults are being tested here. It would be a good idea to test the non-default configurations too